tools/editor-ui/index.html

```diff
@@ -16,54 +16,28 @@
 
 	<body style="font-family:Arial">
 
-	<H2><center>Taxonomy Editor</center></H2>
+	<H2><center>Taxonomy Viewer</center></H2>
```

Comment:
```suggestion
	<H2 style="text-align: center">Taxonomy Viewer</H2>
```

(center should work in all major web browsers but it is deprecated)